File: backend/src/main/java/com/teamj/dto/UserDTO.java
package com.teamj.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@Builder
@RequiredArgsConstructor
public class UserDTO {

    private final Long usersIdx;

    private final String usersNickname;
    
    // private final  String profileImageUrl;

    private final String grade;
}

File: front/lib/dto/user_dto.dart
class UserDto {
  final int usersIdx;
  final String usersNickname;
  final String grade;

  UserDto({required this.usersIdx, required this.usersNickname,required this.grade});

  factory UserDto.fromJson(Map<String, dynamic> json) {
    return UserDto(
      usersIdx: json['usersIdx'],
      usersNickname: json['usersNickname'],
      grade:json['grade'],
    );
  }
}